src/modules/Packages.rb

```diff
@@ -80,6 +80,7 @@ def main
       Yast.import "URL"
       Yast.import "PackagesProposal"
       Yast.import "HTML"
+      Yast.import "NtpClient"
```

Comment:
this adds dependency on ntp client always, which looks wrong. I think you should import it in method itself, so it will be imported only when needed ( during installation ).